new-dti-website/components/sponsor/SponsorshipTable.tsx

```diff
@@ -0,0 +1,110 @@
+import { useState } from 'react';
+import Image from 'next/image';
+import benefitData from './data/benefits.json';
+import medalData from './data/medals.json';
+import useScreenSize from '../../src/hooks/useScreenSize';
+import { TABLET_BREAKPOINT } from '../../src/consts';
+
+type Tier = 'bronze' | 'silver' | 'gold' | 'platinum';
+
+const { medals } = medalData;
+const { benefits } = benefitData;
+
+const tiers = Object.keys(medals);
+
+const SponsorshipTableMobile = () => {
+  const [selectedMedal, setSelectedMedal] = useState<Tier>('bronze');
+  return (
+    <>
+      <div className="flex flex-col gap-7 pb-6 border-b-2 border-black">
+        <h3 className="font-semibold text-2xl">Sponsorship benefits</h3>
+        <div className="flex h-fit justify-between">
+          {Object.keys(medals).map((medal) => (
+            <div className="flex justify-center items-center" key={medal}>
+              <Image
+                src={medals[medal as Tier][selectedMedal === medal ? 'sticker' : 'shadow']}
+                alt={medal}
+                onClick={() => setSelectedMedal(medal as Tier)}
+                height={selectedMedal === medal ? 107 : 87}
```

Comment:
These feel a bit like magic numbers. It would be nice to have them moved out into constants.